includes/Modules/Analytics_4.php

```diff
@@ -1666,23 +1671,9 @@ protected function parse_data_response( Data_Request $data, $response ) {
 			case 'GET:accounts':
 				return array_map( array( self::class, 'filter_account_with_ids' ), $response->getAccounts() );
 			case 'GET:account-summaries':
-				$account_summaries = array_map(
-					function ( $account ) {
-						$obj                    = self::filter_account_with_ids( $account, 'account' );
-						$obj->propertySummaries = array_map( // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
-							function ( $property ) {
-								return self::filter_property_with_ids( $property, 'property' );
-							},
-							$account->getPropertySummaries()
-						);
-
-						return $obj;
-					},
-					$response->getAccountSummaries()
-				);
-				return Sort::case_insensitive_list_sort(
-					$account_summaries,
-					'displayName'
+				return array(
+					'accountSummaries' => $response->getAccountSummaries(),
+					'nextPageToken'    => $response->getNextPageToken(),
```

Comment:
Since the response is essentially unchanged here, let's simply remove the case and return it as-is.